logger/common_others.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// +build !windows
+
+package logger
+
+import (
+	"fmt"
+	"github.com/aws/shim-loggers-for-containerd/debug"
```

Comment:
nit: we are listing imports into three sections:
`golang libraries`, 
`other third-party libraries`,
`libraries inside the shim logger package`

Reply:
I'll change the import order.